Noah bugfixes

noahvaks opened the pull request.

noahvaks committed:
quick fixes

login fields now clear after login attempt
daily sales report now updates on purchase

noahvaks committed:
formatting update

login stays consistent and receipt now shows proper decimal places

noahvaks commented:
Check to see that the login fields don't move when an error message pops up and that the receipt table on POS page has the correct number of decimal places for its fields (using "beef steak, 4" should work to test)

Dogguy0210 commented:
I can confirm the correct number of decimal places is put in, but I don't understand the other thing you want me to test

Dogguy0210 commented:
I approve this pull request

noahvaks committed:
tab order + removed buttons

tab order now should step through program correctly and i've taken out the login override buttons because we're pretty close to finishing the program
